data/5e/wizards_of_the_coast/core/players_handbook/ph_classes.lst

```diff
@@ -542,7 +542,7 @@ CLASS:Warlock	SPELLSTAT:CHA	SPELLTYPE:Arcane	BONUS:CASTERLEVEL|Warlock|CL
 # Class Name
 CLASS:Wizard	HD:6	TYPE:PC.Base	DEFINE:WizardLVL|0	BONUS:VAR|WizardLVL|CL	BONUS:VAR|SpellCasterTotalLVL|WizardLVL	BONUS:VAR|SpellCasterClassCount|1
 CLASS:Wizard	PREMULT:1,[!PRETEMPLATE:1,First Level],[PREABILITY:1,CATEGORY=Internal,Multiclass Allowed ~ Wizard],[PRECLASS:1,Wizard=1]
-CLASS:Wizard	SPELLSTAT:INT	SPELLTYPE:Arcane	BONUS:CASTERLEVEL|Wizard|CL
+CLASS:Wizard	SPELLSTAT:INT	SPELLTYPE:Arcane	MEMORIZE:NO	BONUS:CASTERLEVEL|Wizard|CL
```

Comment:
MEMORIZE:NO is valid only if it is coupled with a KNOWN tag in the class. This change breaks 5e characters.